package.json

```diff
@@ -12,6 +12,8 @@
     "test:watch": "jest --watch",
     "test:e2e": "jest test/e2e --testTimeout=600000 --runInBand --testMatch='**/test/e2e/**/*.test.ts'",
     "test:e2e:rest-api": "jest test/e2e/api-workflow.test.ts --testTimeout=600000 --runInBand --testMatch='**/test/e2e/**/*.test.ts'",
+    "test:e2e:client": "jest test/e2e/client --testTimeout=600000 --runInBand --testMatch='**/test/e2e/client/**/*.test.ts'",
+    "test:e2e:client:initial-connection": "jest test/e2e/client/standards/initial-connection.test.ts --testTimeout=600000 --runInBand --testMatch='**/test/e2e/client/**/*.test.ts'",
```

Comment:
I don't think we need this script.  WE really just want a `test:e2e` that runs everything in e2e that the CI will run. We can and should remove the other scripts.